files/Aboot/boot0.j2

```diff
@@ -122,10 +125,16 @@ platform_specific() {
         aboot_machine=arista_7050_qx32s
         echo "modprobe.blacklist=radeon" >>/tmp/append
     fi
-    if [ "$sid" = "Upperlake" ]; then
+    if [ "$sid" = "Upperlake" ] || [ "$sid" = "UpperlakeES" ]; then
```

Comment:
what is the flash size for upperlakeES? probably, we can increase the varlog size to 200MB?